client/package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
I'm confused: are we using bootstrap instead of material-ui? if so, do we need these dependencies still?

Reply:
Yes we are, unused packages have been deleted.